kernel/classes/datatypes/ezxmltext/ezxmlinputparser.php

```diff
@@ -583,6 +583,59 @@ function parseTag( &$data, &$pos, &$parent )
         Helper functions for pass 1
     */
 
+    /**
+     * Finds the postion of the > character which marks the end of the opening
+     * tag that starts at $tagBeginPos in $data.
+     * It's not as easy as it seems, because some '>' can also appear in attributes.
+     * So we need to iterate over the next '>' characters to find the correct one.
+     * See https://jira.ez.no/browse/EZP-26096
+     *
+     * @param string $data
+     * @param integer $tagBeginPos
+     * @param integer $offset used for recursive call when a > is found in an attribute.
+     * @return integer|false
+     */
+    private function findEndOpeningTagPosition( $data, $tagBeginPos, $offset = 0 )
+    {
+        $endPos = strpos( $data, '>', $tagBeginPos + $offset );
+        if ( $endPos === false )
+        {
+            return false;
+        }
+        $tagCode = substr( $data, $tagBeginPos, $endPos - $tagBeginPos );
+        if ( strpos( $tagCode, '=' ) === false )
+        {
+            // this tag has no attribute, so the next '>' is the right one.
+            return $endPos;
+        }
+        if ( $this->isValidXmlTag( $tagCode ) )
+        {
+            return $endPos;
+        }
+        return $this->findEndOpeningTagPosition( $data, $tagBeginPos, $endPos + 1 );
```

Comment:
You are mixing up the offset with the absolute position. The code should read:
return $this->findEndOpeningTagPosition( $data, $tagBeginPos, $endPos - $tagBeginPos + 1 );